hecuba_py/tests/withQbeast/QbeastIterator_tests.py

```diff
@@ -1,61 +0,0 @@
-import unittest
```

Comment:
Can we adapt the Qbeast tests to the new implementation?
At least the "rebuild" test where a getID is performed should be kept